### PR TITLE
3/edit notes

### PR DESCRIPTION
**Pull request by hagoptaminian**
@juancarlosfarah check it out so far. No need for a code review if you're busy but let me know if you have any comments playing around with it. Looks like this:

Screen Shot 2021-06-16 at 3 58 56 PM

### CODE REVIEW

**Comment by pyphilia** [comment]
That looks so nice! I played around a bit with it, some minor comments:
- I would change the pointer cursor (when clicking to add a note) with a + sign
- Having a long description triggers a scroll, it would be ideal to be able to resize notes to see everything.

Otherwise really good job!!

**Comment by juancarlosfarah** [comment]
@hagoptaminian. This looks great! I agree with Kim's first comment. Regarding the second one, I'm not sure. Maybe we can discuss that tomorrow.

👏

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Very nice! Clean code and great first version. Just left some comments and questions in there. Can address here or in future PRs if you've already planned it.

A couple general comments.

1. How about using the different variants of `Typography` instead of changing the font manually? 
2. There's some repeated code, specially in the styling and some minor helper functions. Maybe factor that out.

Otherwise it looks good! Nice job.

File: src/components/App.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ export class App extends Component {
     lang: PropTypes.string,
     mode: PropTypes.string,
     view: PropTypes.string,
-    headerVisible: PropTypes.bool.isRequired,
+    // headerVisible: PropTypes.bool.isRequired,
```
[the remark this answers is not in the document]
Comment (hagoptaminian) [reply src/components/App.js:26]: Good catch. Indeed I've opted to go with no header for this application.

File: src/components/main/Canvas.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  return (
+    <div className={classes.mainContainer} onClick={handleCanvasClick}>
+      {notes.map((note, index) => (
+        // eslint-disable-next-line react/no-array-index-key
+        <Note note={note} key={index} />
+      ))}
+      {activeFormExists && <Form />}
+      <img
+        src="https://images.unsplash.com/photo-1527245592484-dff09c8e4cd9?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1500&q=80"
```
[the remark this answers is not in the document]
Comment (hagoptaminian) [reply src/components/main/Canvas.js:60]: TBD! For now I left the default as a lightly colored salmon div/background.

File: src/components/main/form/FormActions.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+const useStyles = makeStyles(() => ({
+  iconContainer: {
+    display: 'flex',
+    justifyContent: 'flex-end',
+    alignItems: 'center',
+  },
+  formIcon: {
+    color: '#484848',
+    textAlign: 'right',
+    fontSize: '1.1vw',
```
[the remark this answers is not in the document]
Comment (hagoptaminian) [reply src/components/main/form/FormActions.js:20]: To make text responsive, I need to use either `vw` or `vh`. I'm choosing the numbers by eyeballing...

**Comment by hagoptaminian** [comment]
Thanks @pyphilia @juancarlosfarah. Changing the icon to a plus seems to be the way to go. Updating this PR with another (#11) to avoid rebasing.

**Pull request closed without merging**